runtimes/package.json

```diff
@@ -53,7 +53,8 @@
         "rxjs": "^7.8.2",
         "vscode-languageserver": "^9.0.1",
         "vscode-languageserver-protocol": "^3.17.5",
-        "win-ca": "^3.5.1"
+        "win-ca": "^3.5.1",
+        "@aws/mynah-ui": "^4.31.0-beta.6"
```

Comment:
Flare protocol is UI-agnostic, we should not depend on Mynah-specific types in protocol.